marlowe-chain-sync/deploy/tx.sql

```diff
@@ -0,0 +1,51 @@
+-- Deploy chain:tx to pg
+-- requires: block
+
+BEGIN;
+
+SET client_min_messages = 'warning';
+
```

Comment:
Is the lack of primary key, foreign key, and check constraints here (and throughout the schema) for optimization?

Reply:
It's a necessary tradeoff for acceptable insert performance. It is justified because it is a copy of a verified data source (the node already enforces constraints), so we can afford to be more relaxed by avoiding UNIQUE, PRIMARY KEY, and FOREIGN KEY constraints.